pwndbg/strings.py

```diff
@@ -31,7 +31,13 @@ def update_length():
     message = gdb.execute('show print elements', from_tty=False, to_string=True)
     message = message.split()[-1]
     message = message.strip('.')
-    length  = int(message)
+    try:
+        length  = int(message)
+    except ValueError:
+        if message == 'unlimited:
```

Comment:
```SyntaxError: EOL while scanning string literal```
I have the feeling this code wasn't tested much :stuck_out_tongue_closed_eyes: 

it would make more sense to first check for 'unlimited' before trying to make an integer out of it. I don't think the try-except is needed at all